posts/zzz_DO_NOT_EDIT_pharmaverse.../pharmaverse__examples.qmd

```diff
@@ -0,0 +1,93 @@
+
+The site is organized into easy-to-navigate sections:
+
+-   **SDTM** examples like DM (Demographics), VS (Vital Signs), AE (Adverse Events).
+
+-   **ADaM** examples including ADSL (Subject Level), ADTTE (Time-to-Event) and ADVS (Vital Signs).
+
+-   **TLG** examples showing how to build demographic tables, adverse event summaries, and pharmacokinetic displays.
+
+-   **Interactive** content such as data visualization apps built with teal.
+
+-   **Automated reporting** of outputs via slides or docs.
+
+-   **eSubmission** for submission preparation.
+
+Each example serves as a mini-tutorial with code snippets walking you through steps and letting you see how packages can be used in a realistic workflow.
+
+### 💬 Why It Matters
+
+Open source thrives on collaboration, and the pharmaverse ecosystem embodies this by **bringing developers and users together** to share tools, workflows, and best practices. The Examples site is a bridge between learning and doing—it helps users:
+
+-   Understand how packages fit together
+
+-   Forum to share how packages are used by different teams
+
+-   Work towards reproducible, end to end workflows
+
+All of this contributes to **faster, more reliable clinical reporting** and supports the broader mission of shared problem solved across the industry.
+
+### 🤝 How You Can Contribute
+
+The pharmaverse examples site is a **community-driven site** that grows with contributions from developers and practitioners.
+
+Here’s how you can get involved:
+
+#### 📌 1. **Contribute an Example**
+
+Have a example that others might benefit from? Add a new example by contributing via the [GitHub repo](https://github.com/pharmaverse/examples). The source code includes guidance in the README about how to structure and submit contributions.
+
+Alternatively, if you have identified an example that you'd like to see added then please add an [issue](https://github.com/pharmaverse/examples/issues)
+
+#### 🛠 2. **Improve Existing Content**
+
+All contributions are welcomed no matter how big or small. Please feel free to suggest edits to our supporting docs and code snippets either via creating an [issue](https://github.com/pharmaverse/examples/issues) and / or making the suggested changes via a [pull request](https://github.com/pharmaverse/examples).
+
+#### 💬 3. **Engage With the Community**
+
+Join the pharmaverse community on [Slack](https://pharmaverse.slack.com/) participate in discussions, and help others get up to speed. Contributions such as feedback, guidance, and engaging your teams with these examples are equally valuable.
+
+✨ Wrapping up Whether you’re just starting with pharmaverse packages or looking to deepen your practice, we hope the pharmaverse examples site provides you with valuable content to explore.
+
+Start exploring today, and maybe even contribute your first example!
```

Comment:
Should we say anything about being a small sub-team of pharmaverse from people across many companies who meet monthly and they could also feel free to join this group? Might attract more people if they see we're a connected group that they could be part of